### PR TITLE
Remove deprecated options: transform and tombstone

### PR DESCRIPTION
**Pull request by acoburn**
Resolves: https://jira.duraspace.org/browse/FCREPO-2209

### CODE REVIEW

File: src/test/java/org/fcrepo/camel/FcrepoProducerTest.java [hunk 1]
```diff
@@ -707,58 +526,6 @@ public void testGetSecureProducer() throws Exception {
     }
 
     @Test
-    public void testGetSecureProducerWithScheme() throws Exception {
```
Comment (awoods) [inline src/test/java/org/fcrepo/camel/FcrepoProducerTest.java]: This method seems unrelated to `transform` and `tombstone`. Was it intentionally removed?

Comment (acoburn) [inline src/test/java/org/fcrepo/camel/FcrepoProducerTest.java]: Good catch -- this test method should remain.
